### PR TITLE
Fix comments

### PR DESCRIPTION
**Pull request by TranLeKhanhDuyen**

### CODE REVIEW

File: practice-two/src/styles/pages/hero.css [hunk 1]
```diff
@@ -115,27 +110,22 @@ input:focus {
...
+    padding: 7px 31px 13px 15px;
   }
 
   .hero-button {
     margin: 0;
-    font-size: 18px;
     letter-spacing: -0.36px;
   }
 
   .text-access {
```
Comment (tuanthaiquang-agilityio) [inline practice-two/src/styles/pages/hero.css:122]: Why text-access here?

File: practice-two/src/styles/components/text.css [hunk 2]
```diff
@@ -27,3 +27,42 @@ li {
...
+
+.text-lg {
+  font-size: 24px;
+  line-height: 34px;
+}
+
+.text-xl {
+  font-size: 36px;
+  line-height: 34px;
+  font-weight: var(--fw-bold);
```
Comment (tuanthaiquang-agilityio) [inline practice-two/src/styles/components/text.css:50]: .text class? No need to use .text class?

File: practice-two/src/styles/pages/hero.css (hunk 1 above)
Comment (TranLeKhanhDuyen) [reply practice-two/src/styles/pages/hero.css:122]: update: this font-size removed

File: practice-two/src/styles/components/text.css (hunk 2 above)
Comment (TranLeKhanhDuyen) [reply practice-two/src/styles/components/text.css:50]: to reuse the same texts

**Pull request merged**